doc/run-cloud-build-locally.md

```diff
@@ -0,0 +1,197 @@
+# Running `cloudbuild.yaml` Locally with Docker
+
+`cloudbuild-local` is deprecated; however you can still iterate on the
+`cloudbuild.yaml` steps by running them manually using Docker and the same
+tooling that Cloud Build uses. This document provides a lightweight, repeatable
+workflow for running the CI pipeline locally while being able to iterate
+quickly.
+
+## Prerequisites
+
+- Docker Desktop (with `docker compose`) installed and running.
+- A local JDK 17 + Maven 3.8.x for the Maven build steps.
+- `python3` (for `synthea-uploader` unit tests) and `python -m unittest`
+  available on `$PATH`.
+- Access to the workspace root (same directory that holds `cloudbuild.yaml`).
```

Comment:
Should this simply be: "The GitHub repo is checked out locally" or something like that?